assets/xml/archives/icon_item_static.xml

```diff
@@ -1,194 +1,222 @@
+        <Texture Name="gPauseMasks04Tex" OutName="pause_masks_0_4" Format="ia8" Width="80" Height="32" Offset="0x8A640" />
+        <Texture Name="gPauseMasks14Tex" OutName="pause_masks_1_4" Format="ia8" Width="80" Height="32" Offset="0x8B040" />
+        <Texture Name="gPauseMasks24Tex" OutName="pause_masks_2_4" Format="ia8" Width="80" Height="32" Offset="0x8BA40" />
+        <Texture Name="gPauseSelectItem04Tex" OutName="pause_select_item_0_4" Format="ia8" Width="80" Height="32" Offset="0x8C440" />
+        <Texture Name="gPauseSelectItem14Tex" OutName="pause_select_item_1_4" Format="ia8" Width="80" Height="32" Offset="0x8CE40" />
+        <Texture Name="gPauseSelectItem24Tex" OutName="pause_select_item_2_4" Format="ia8" Width="80" Height="32" Offset="0x8D840" />
+        <Texture Name="gPauseMap04Tex" OutName="pause_map_0_4" Format="ia8" Width="80" Height="32" Offset="0x8E240" />
+        <Texture Name="gPauseMap14Tex" OutName="pause_map_1_4" Format="ia8" Width="80" Height="32" Offset="0x8EC40" />
+        <Texture Name="gPauseMap24Tex" OutName="pause_map_2_4" Format="ia8" Width="80" Height="32" Offset="0x8F640" />
+        <Texture Name="gPauseQuestStatus04Tex" OutName="pause_quest_status_0_4" Format="ia8" Width="80" Height="32" Offset="0x90040" />
+        <Texture Name="gPauseQuestStatus14Tex" OutName="pause_quest_status_1_4" Format="ia8" Width="80" Height="32" Offset="0x90A40" />
+        <Texture Name="gPauseQuestStatus24Tex" OutName="pause_quest_status_2_4" Format="ia8" Width="80" Height="32" Offset="0x91440" />
+        <Texture Name="gPauseSave04Tex" OutName="pause_save_0_4" Format="ia8" Width="80" Height="32" Offset="0x91E40" />
+        <Texture Name="gPauseSave14Tex" OutName="pause_save_1_4" Format="ia8" Width="80" Height="32" Offset="0x92840" />
+        <Texture Name="gPauseSave24Tex" OutName="pause_save_2_4" Format="ia8" Width="80" Height="32" Offset="0x93240" />
+
+        <!-- Textures used by display lists extracted from icon_item_vtx_static -->
+        <Texture Name="gABtnSymbolTex" OutName="a_button_symbol" Format="ia8" Width="24" Height="16" Offset="0x93C40"/>
+        <Texture Name="gBBtnSymbolTex" OutName="b_button_symbol" Format="ia8" Width="24" Height="16" Offset="0x93DC0"/>
+        <Texture Name="gCBtnSymbolsTex" OutName="c_button_symbols" Format="ia8" Width="48" Height="16" Offset="0x93F40"/>
+        <Texture Name="gNamePanelLeftTex" OutName="name_panel_left" Format="ia8" Width="72" Height="24" Offset="0x94240"/>
+        <Texture Name="gNamePanelRightTex" OutName="name_panel_right" Format="ia8" Width="72" Height="24" Offset="0x94900"/>
+        <Texture Name="gZButtonTex" OutName="z_button" Format="ia8" Width="24" Height="32" Offset="0x94FC0"/>
+        <Texture Name="gRButtonTex" OutName="r_button" Format="ia8" Width="24" Height="32" Offset="0x952C0"/>
+        <Texture Name="gUnknownJPTex" OutName="unknown_JP" Format="ia8" Width="80" Height="16" Offset="0x955C0"/>
```

Comment:
More out of curiosity than anything else, what's unknown about it?

Reply:
```suggestion
        <Texture Name="gUnknownJPNTex" OutName="unknown_jp" Format="ia8" Width="80" Height="16" Offset="0x955C0"/>
```
Seems to be how it is done elsewhere.

Reply:
Not a clue. Probably the translation? Just copied the name from OoT. 

![image](https://github.com/zeldaret/mm/assets/47598039/d835c7c6-409b-424a-b911-ab10bd64d03f)

Reply:
I don't know if you want to update it, considering it came from OoT, but it apparently says something like "Show a musical note".

Reply:
Oh nice, yeah I'll change it. Currently having local build issues so will get to it soon :tm: